include/libdnf-cli/argument_parser.hpp

```diff
@@ -607,6 +607,10 @@ class ArgumentParser {
     /// Gets a list of all positional argument stored in the argument parser.
     const std::vector<std::unique_ptr<PositionalArg>> & get_positional_args() const noexcept { return pos_args; }
 
+    /// Prints a completed argument from the `Complete_arg_idx` position or a table with suggestions and help
+    /// to complete if there is more than one solution.
```

Comment:
`Complete_arg_idx` should be with lowercase `c`.

The doctext is a bit hard to understand, what about:
> Prints **the** completed argument from the `complete_arg_idx` position in the `argv` array. In case of more than one potential completion matches, prints a table with the potential matches along with their short help descriptions.

Reply:
Thanks. Your text is easier to understand. I used it.